### PR TITLE
Make the oc task reusable

### PR DESCRIPTION
**Pull request by piyush-garg**
This will add a script to Dockerfile which will
receive command as a string and then parse them
to execute with OC

This way we can make the task reusable

Changes to use the Ubi image as the base image

Update both the examples to use reusables task

Update README

### CODE REVIEW

File: openshift-cli/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,16 @@
-FROM docker.io/centos:7
+FROM registry.access.redhat.com/ubi8-dev-preview/ubi:latest
```
Comment (siamaksade) [inline openshift-cli/Dockerfile]: The UBI 8 GA should be used instead: `registry.access.redhat.com/ubi8/ubi`

File: openshift-cli/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 RUN cd /tmp \
   && yum -y update \
   && yum install -y wget \
   && wget https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz \
   && tar -xvzf oc.tar.gz \
-  && mv oc /usr/local/bin/ \
+  && mv oc oc-origin \
```
Comment (vdemeester) [inline openshift-cli/Dockerfile]: This could be done in one shot : `mv oc /usr/local/bin/oc-origin` :wink:

**Comment by siamaksade** [comment]
Is there a reason for using `openshift-cli` for the task name and directory but `openshift-client` in the YAML file names?

**Comment by piyush-garg** [comment]
@siamaksade There was no reason for that, I have updated all to `openshift-client` for consistent naming

**Pull request merged**